### ISSUE TITLE
Batch Insert Support

### ISSUE THREAD

**Issue by irakitin** [sorm/sorm#54]
Haven't found any information on support for batch inserts. Is it available/planned?

**Comment by nikita-volkov** [comment]
No. It's not available, and unfortunately currently I'm not working on any new features for SORM. Contribution is welcome though.
